pkg/apis/ignite/types.go

```diff
@@ -38,17 +38,6 @@ const (
 	ImageSourceTypeDocker ImageSourceType = "Docker"
 )
 
-// OCIImageClaim defines a claim for importing an OCI image
-type OCIImageClaim struct {
-	// Type defines how the image should be imported
-	Type ImageSourceType `json:"type"`
```

Comment:
Also remove the `ImageSourceType` enum

Reply:
Removed in 9d6ab9e.

Reply:
Also removed `GetImageSourceTypes` as it's redundant.